common/test/acceptance/tests/lms/test_lms_courseware_search.py

```diff
@@ -68,8 +68,10 @@ def _auto_auth(self, username, email, staff):
         Logout and login with given credentials.
         """
         LogoutPage(self.browser).visit()
-        AutoAuthPage(self.browser, username=username, email=email,
-                     course_id=self.course_id, staff=staff).visit()
+        login_page = AutoAuthPage(self.browser, username=username, email=email,
```

Comment:
I'm surprised that we needed this - was this just a "try-this" kind of thing, or did it actually fix a problem for you on your machine?